dolphinscheduler-tools/src/main/bin/create-schema.sh

```diff
@@ -18,7 +18,8 @@
 
 BIN_DIR=$(dirname $0)
 DOLPHINSCHEDULER_HOME=${DOLPHINSCHEDULER_HOME:-$(cd $BIN_DIR/../..; pwd)}
-source "$DOLPHINSCHEDULER_HOME/tools/bin/dolphinscheduler_env.sh"
+
+source "$DOLPHINSCHEDULER_HOME/bin/env/dolphinscheduler_env.sh"
```

Comment:
if we do not use `dolphinscheduler_env.sh` file anymore, we should also remove the assembly config https://github.com/apache/dolphinscheduler/blob/de50f43de63e42488cb38169cd56fba77dd24fc6/dolphinscheduler-tools/src/main/assembly/dolphinscheduler-tools.xml#L41-L49

Reply:
I think it is a good idea to remove the assembly config in case users get confused. Will fix it in the next commit. : )

Reply:
@SbloodyS @zhongjiajie E2E tests failed. May I ask is it caused by the removal of assembly config? Is there any way to bypass it? Thx!

Reply:
I do not think it will cause the failed. I will restart the CI

Reply:
@EricGao888 It's E2E build failed. Maybe it's the problem with GitHub Actions since it can't guarantee 99.99% stability...

Reply:
> @EricGao888 It's E2E build failed. Maybe it's the problem with GitHub Actions since it can't guarantee 99.99% stability...

could I @github at this time？ 🤣

Reply:
@github +1